pkg/kubelet/prober/prober.go

```diff
@@ -68,18 +68,8 @@ func newProber(
 	}
 }
 
-// recordContainerEvent should be used by the prober for all container related events.
-func (pb *prober) recordContainerEvent(pod *v1.Pod, container *v1.Container, eventType, reason, message string, args ...interface{}) {
-	ref, err := kubecontainer.GenerateContainerRef(pod, container)
-	if err != nil {
-		klog.ErrorS(err, "Can't make a ref to pod and container", "pod", klog.KObj(pod), "containerName", container.Name)
-		return
-	}
-	pb.recorder.Eventf(ref, eventType, reason, message, args...)
-}
```

Comment:
I'm not sure I fully understand the idea of removing this method. worker.go is in the same package, so it can call this API without this modification.